### PR TITLE
Add support for make-latest to create_git_release and create_git_tag_and_release

### PR DESCRIPTION
**Pull request by SebastienSyd**
### The problem:

def "create_git_release" and "create_git_tag_and_release" can not create releases that are not the "latest", as by default they are set as "latest"

### How solving the problem:

adding optional parameter "make_latest", from original api doc: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release

similar to: https://github.com/PyGithub/PyGithub/pull/2888

### CODE REVIEW

File: github/Repository.py [hunk 1]
```diff
@@ -1264,6 +1264,7 @@ def create_git_tag_and_release(
         draft: bool = False,
         prerelease: bool = False,
         generate_release_notes: bool = False,
+        make_latest: str = "true",
```
Comment (EnricoMi) [inline github/Repository.py:1267]: This should be a `bool`:
```suggestion
        make_latest: bool = True,
```

Comment (EnricoMi) [inline github/Repository.py:1267]: What is the current behaviour of Github with `make_latest` omitted?

File: tests/Repository.py [hunk 2]
```diff
@@ -432,6 +432,7 @@ def testCreateGitReleaseWithAllArguments(self):
             True,
             False,
             "da9a285fd8b782461e56cba39ae8d2fa41ca7cdc",
+            "true",
         )
         self.assertEqual(release.tag_name, "vX.Y.Z-by-PyGithub-acctest2")
```
Comment (EnricoMi) [inline tests/Repository.py:437]: Does `Release` have an attribute like `latest`?

**Review by EnricoMi — CHANGES_REQUESTED** [review]

File: github/Repository.py (hunk 1 above)
Comment (SebastienSyd) [reply github/Repository.py:1267]: according to the official documentation, the default is "true"

File: tests/Repository.py (hunk 2 above)
Comment (SebastienSyd) [reply tests/Repository.py:437]: I haven't found any `latest` attribute sadly

File: github/Repository.py (hunk 1 above)
Comment (EnricoMi) [reply github/Repository.py:1267]: alright

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Comment by SebastienSyd** [comment]
@EnricoMi any news?

File: github/Repository.py (hunk 1 above)
Comment (bebach) [reply github/Repository.py:1272]: Why should make_latest be a string and not a bool?

**Comment by bebach** [comment]
> @EnricoMi any news?

Good point. This would be mighty helpful for us :-)

**Pull request merged**